docs/agents/sematext-agent/windows-installation.md

```diff
@@ -0,0 +1,155 @@
+3. Follow the installation steps and accept the license agreement 
+4. Set the Infra App token
+5. Select a region (EU or US) based on where you created your account in Sematext Cloud
+
+After completing these steps, Sematext Agent will be automatically started as a Windows Service.
+
+## Install from command line
+
+Download the [Sematext Agent installer](https://todo-add-public-installer-repo).
+
+### Using PowerShell
+
+Run the following command as Administrator from the folder that you downloaded the installer:
+
+```
+Start-Process -Wait msiexec -ArgumentList "/qn /i $($msiFileName = 'sematext-agent-VERSION.msi'; $msiFileName) /L*V `"$msiFileName.log`" INFRA_APP_TOKEN=YOUR_INFRA_TOKEN REGION=YOUR_REGION_EU_or_US"
+```
+
+Where:
+
+- `YOUR_INFRA_APP_TOKEN`: the Infra App token
+- `YOUR_REGION_EU_or_US`: your Sematext Cloud region
+- `VERSION`: the agent version
+
+For example:
+
+```
+Start-Process -Wait msiexec -ArgumentList "/qn /i $($msiFileName = 'sematext-agent-3.3.0.2.msi'; $msiFileName) /L*V `"$msiFileName.log`" INFRA_APP_TOKEN=7511db7f-c060-4e10-b667-5f2653d4933e REGION=EU"
+```
+
+### Using Windows CMD:
+
+Run the following command as Administrator from the folder that you downloaded the installer:
+
+```
+start /wait msiexec /qn /i "sematext-agent-VERSION.msi" /L*V "sematext-agent-VERSION.msi.log" INFRA_TOKEN=YOUR_INFRA_APP_TOKEN REGION=YOUR_REGION_EU_or_US
+```
+
+Where:
+
+- `YOUR_INFRA_APP_TOKEN`: the Infra App token
+- `YOUR_REGION_EU_or_US`: your Sematext Cloud region
+- `VERSION`: the agent version
+
+For example:
+
+```
+start /wait msiexec /qn /i "sematext-agent-3.3.0.2.msi" /L*V "sematext-agent-3.3.0.2.msi.log" INFRA_APP_TOKEN=7511db7f-c060-4e10-b667-5f2653d4933e REGION=EU
+```
+
+After completing these steps, Sematext Agent will be automatically started as a Windows Service.
+
+## How to check the Sematext Agent service status
+
+### Windows Services
+
+Open Windows Services and find the Sematext Agent service:
+
+![Sematext Agent service](images/sematext-agent-service.png)
+
+### Command line
+
+Run the following command:
+
+```
+cd  'C:\Program Files\Sematext Agent\' .\st-agent-amd64.exe windows-service status
+```
+
+## How to start / stop / restart Sematext Agent
+
+To start, stop or restart the Sematext Agent, you can perform these actions through the GUI:
+
+- Start: Open the Windows Services application by searching for "Services" in the Windows search bar, locate "Sematext Agent" in the list of services, right-click on it, and select "Start"
+- Stop: Similarly, in the Services application, locate "Sematext Agent", right-click on it, and select "Stop"
+- Restart: To restart the Sematext Agent, you can either stop and then start it again using the aforementioned steps or right-click on "Sematext Agent" in the Services application and select "Restart"
+
+Alternatively, you can use the command-line interface by running one of the following commands:
+
+Start:
+
+```
+cd 'C:\Program Files\Sematext Agent\'
```

Comment:
Looks like All Windows Service commands require Admin perm. See, I can't even run `status` command.

![Screenshot from 2024-04-24 18-36-28](https://github.com/sematext/docs/assets/20258973/85d1761c-553e-4ff2-8f01-63cd144ce1cc)

But I can run it if I have admin perms. 
![Screenshot from 2024-04-24 18-37-37](https://github.com/sematext/docs/assets/20258973/45474c91-4c9f-483e-b455-e422b978bc2e)

So they need to run their PowerShell with admin perms. I guess they would know "how-to".

Reply:
I think it should be ok. We can add `-Verb RunAs` at the end of each command but it may be too much.

Reply:
updated description